cloud_connectors/azure/mqtt_connector/azure_function/src/run.cs

```diff
@@ -57,11 +69,14 @@ public MQTTConnectorAzureFunction(ILogger<MQTTConnectorAzureFunction> logger)
         /// </summary>
         /// <param name="dataTypeName">the name of the data type.
         /// <returns>Returns a task for updating a digital twin instance.</returns>
```

Comment:
Maybe add a comment for `NotSupportedException`.

```suggestion
        /// <exception cref="NotSupportedException">Thrown if the data type is not supported.</exception>
       /// <returns>Returns a task for updating a digital twin instance.</returns>
```